### PR TITLE
feat: audit request post

### PR DESCRIPTION
**Pull request by kristovatlas**
# Description:
This PR introduces the blog post "Call for Security Auditors - gno.land Audit". It is written by @kristovatlas on 2024/11/27.

### CODE REVIEW

File: posts/2024-11-28_audit-request/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+title: "Call for Security Auditors - gno.land Audit"
+publication_date: 2024-11-28T00:00:00Z
+slug: audit-proposal-request
+tags: [gnoland, gnovm, security, audit]
+authors: [kristovatlas]
+---
+
+# Call for Security Auditors - gno.land Audit
+
+In the gno.land ecosystem, security is one of our top priorities as we continue to develop our blockchain focused on smart contracts and decentralized applications. As we move closer to the mainnet launch, we're calling on experienced security auditors to help us ensure the robustness and security of our codebase.
```
Comment (Kouteki) [inline posts/2024-11-28_audit-request/README.md:11]:
```suggestion
In the gno.land ecosystem, security is one of our top priorities as we continue to develop our blockchain focused on smart contracts and decentralized applications. As we move closer to the beta launch, we're calling on experienced security auditors to help us ensure the robustness and security of our codebase.
```

**Review by Kouteki — APPROVED** [review]
LGTM, just added a single change

File: posts/2024-11-28_audit-request/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+---
+title: "Call for Security Auditors - gno.land Audit"
```
Comment (leohhhn) [inline posts/2024-11-28_audit-request/README.md:2]:
```suggestion
```

File: posts/2024-11-28_audit-request/README.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+publication_date: 2024-11-28T00:00:00Z
```
Comment (leohhhn) [inline posts/2024-11-28_audit-request/README.md:3]:
```suggestion
publication_date: 2024-12-06T00:00:00Z
```

File: posts/2024-11-28_audit-request/README.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+* **Fuzzing efforts** -- auditors are encouraged to include fuzzing as part of their code review process, though it should be listed as a separate line item.
+
+Please include the timelines for auditing and your current availability beginning December 23rd. 
+
+## Timeline & Contact Information
+We expect the audit to consume at least 4 engineer weeks and conclude by the end of the day January 20th, giving us a week to remediate any high- or critical-severity issues by January 28th. Audit teams may decide to dedicate multiple auditors in parallel to meet our desired timeline. We are open to discussions with auditing teams to answer any questions about the scope of the project.
+
+Please send your proposals via email to **security [at] tendermint [dot] com**. We are happy to meet with potential auditing teams to further discuss the details and answer any questions.
+
+---
```
Comment (leohhhn) [inline posts/2024-11-28_audit-request/README.md:72]:
```suggestion
```

File: posts/2024-11-28_audit-request/README.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    Although this is a lower priority than some previously mentioned, auditors will need to review the gnokey package, which handles key generation and signing, to ensure that security best practices are being followed; for example, ensure our Ledger hardware wallet integration with gnokey uses the correct build flags.
+
+## How to Submit a Proposal
+
+We're looking for proposals that include the following line items:
+
+* **Cost of auditing the entire "gno" repository** -- covering all directories and files in scope.
+* **Cost of auditing the Gno Virtual Machine (gnovm)** -- focused on the **gnovm** subdirectory.
+* **Cost of auditing the other components** -- covering the **gno.land** and **tm2** subdirectories.
+* **Fuzzing efforts** -- auditors are encouraged to include fuzzing as part of their code review process, though it should be listed as a separate line item.
```
Comment (leohhhn) [inline posts/2024-11-28_audit-request/README.md:63]:
```suggestion
- **Cost of auditing the entire "gno" repository** -- covering all directories and files in scope.
- **Cost of auditing the Gno Virtual Machine (gnovm)** -- focused on the **gnovm** subdirectory.
- **Cost of auditing the other components** -- covering the **gno.land** and **tm2** subdirectories.
- **Fuzzing efforts** -- auditors are encouraged to include fuzzing as part of their code review process, though it should be listed as a separate line item.
```

File: posts/2024-11-28_audit-request/README.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
+---
+publication_date: 2024-12-06T00:00:00Z
```
Comment (leohhhn) [inline posts/2024-11-28_audit-request/README.md:2]:
```suggestion
publication_date: 2024-12-05T00:00:00Z
```

**Pull request merged**